docs/index.asciidoc

```diff
@@ -32,8 +32,8 @@ Logstash ships with about 120 patterns by default. You can find them here:
 <https://github.com/logstash-plugins/logstash-patterns-core/tree/master/patterns>. You can add
 your own trivially. (See the `patterns_dir` setting)
 
-If you need help building patterns to match your logs, you will find the
-<http://grokdebug.herokuapp.com> and <http://grokconstructor.appspot.com/> applications quite useful!
+If you need help building patterns to match your logs, you will find the Grok Debugger in Kibana
```

Comment:
@karenzone what's the best way to link to "Grok Debugger in Kibana" at https://www.elastic.co/guide/en/kibana/current/xpack-grokdebugger.html here?

Reply:
If you need help building patterns to match your logs, try the {kibana-ref}/xpack-grokdebugger.html[Grok debugger] in {kib}.

Reply:
Thank you!